src/styles/_display.scss

```diff
@@ -71,3 +71,18 @@
 		padding-left: 5px * $i;
 	}
 }
+
+//----------------------------------------
+// Positioning
+//----------------------------------------
+.pull-left {
```

Comment:
@jrassa do we need these, or are they dupes from what bootstrap provides?

Reply:
As far as I can tell, `pull-left` and `pull-right` aren't coming from bootstrap.  They are currently coming from font-awesome, but they are marked as deprecated and will be removed in v5.  It also just seemed improper to rely on a class from font-awesome for general styling.

Reply:
https://getbootstrap.com/docs/4.0/migration/#utilities

Bootstrap v4 moved to "float-left" and "float-right".

Reply:
Thanks @reblace. I added #30 to cover that migration.